cpu/stm32f3/cpu.c

```diff
@@ -21,6 +22,30 @@
 #include "cpu.h"
 #include "periph_conf.h"
 
+/* Check the source to be used for the PLL */
+#if defined(CLOCK_HSI) && defined(CLOCK_HSE)
+#error "Only provide one of two CLOCK_HSI/CLOCK_HSE"
+#elif CLOCK_HSI
+#define CLOCK_CR_SOURCE             RCC_CR_HSION
+#define CLOCK_CR_SOURCE_RDY         RCC_CR_HSIRDY
+#ifdef RCC_CFGR_PLLSRC_HSI_DIV2
```

Comment:
Not much to do about it except to not automate it and make the user to through all the hussle :p